daemon/cmd/hubble.go

```diff
@@ -106,6 +107,12 @@ func (d *Daemon) launchHubble() {
 			logger.WithError(err).Warn("Failed to initialize Hubble metrics server")
 			return
 		}
+
+		opt := observeroption.WithOnDecodedFlowFunc(func(ctx context.Context, flow *flowpb.Flow) (bool, error) {
+			metrics.ProcessFlow(ctx, flow)
```

Comment:
not related to this particular PR, but it's a little fishy this doesn't return `error`